AppTests/App/Logic/AnalyticsLogicTests.swift

```diff
@@ -44,7 +44,7 @@ extension AnalyticsLogicTests {
 
     XCTAssertEqual(dispatchInterceptor.dispatchedItems.count, 1)
 
-    try XCTAssertType(dispatchInterceptor.dispatchedItems.first, Logic.Analytics.SendOperationalInfoWithExposureIfNeeded.self)
+    try XCTAssertType(dispatchInterceptor.dispatchedItems.first, Logic.Analytics.StochasticallySendOperationalInfoWithExposure.self)
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: AppTests/App/Logic/AnalyticsLogicTests.swift;
  Line: 47;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 132 characters (line_length)